config/application.rb

```diff
@@ -14,5 +14,7 @@ class Application < Rails::Application
 
     # For adding subdirectories
     config.autoload_paths += Dir[Rails.root.join('app', 'models', '{**}')]
+  	# Force lowercase for Acts on Taggable
+  	ActsAsTaggableOn.force_lowercase = true
```

Comment:
Let's put all of these in its own initializer file called `acts_as_taggable_on.rb`